bftengine/src/bcstatetransfer/SourceSelector.cpp

```diff
@@ -160,11 +169,13 @@ SourceReplacementMode SourceSelector::shouldReplaceSource(uint64_t currTimeMilli
       LOG_INFO(logger_,
                "Should replace source: source replacement timeout:" << KVLOG(
                    currentReplica_, dt, sourceReplacementTimeoutMilli_));
+      metrics_.replacement_due_to_periodic_change_++;
       return SourceReplacementMode::GRACEFUL;
     }
   }
   if ((lastInBatch) and (currentReplica_ == currentPrimary_)) {
     LOG_INFO(logger_, "Current replica has become primary" << KVLOG(currentReplica_, currentPrimary_));
+    metrics_.replacement_due_to_source_same_as_primary_++;
```

Comment:
can you please also change the above log, to keep same style as all logs in the function:
`"Should replace source: current replica has become primary:"`